client/components/Nav/NavDropdownMenu.jsx

```diff
@@ -22,8 +23,28 @@ export function useMenuProps(id) {
 function NavDropdownMenu({ id, title, children }) {
   const { isOpen, handlers } = useMenuProps(id);
 
+  let floatingProps;
+  if (id === 'file' || id === 'edit' || id === 'sketch' || id === 'help') {
+    floatingProps = {
+      whileElementsMounted: autoUpdate,
```

Comment:
We need to be aware of this warning from [the docs](https://floating-ui.com/docs/useFloating#whileelementsmounted).

> Warning
> Only use this option if your floating element is conditionally rendered, not hidden with CSS. Use an effect to call and clean up `autoUpdate` in the latter scenario.

We are currently using CSS to show and hide the menu:
https://github.com/processing/p5.js-web-editor/blob/f21b84772c85927f8016d7c23aef7ad1a5a41b64/client/styles/components/_nav.scss#L108-L114

I personally don't like using CSS to hide things so I would be fine if we switched to conditional rendering and kept the `autoUpdate`.

Reply:
conditional rendering would be a better option.

Reply:
`{isOpen && (
        <ul
          className="nav__dropdown"
          ref={refs.setFloating}
          style={floatingStyles}
        >
          <ParentMenuContext.Provider value={id}>
            {children}
          </ParentMenuContext.Provider>
        </ul>
      )}`

conditionally rendering working, but the `nav_dropdown-item` is not navigatting.
@lindapaiste can you please help me with this...

Reply:
@lindapaiste any suggestions on this one